src/RestControllers/SMART/SMARTConfigurationController.php

```diff
@@ -0,0 +1,151 @@
+<?php
```

Comment:
license, author, copyright header

Reply:
disregard above, i see it is already there below :)

Reply:
I think I'm going to have to figure out if phpstorm can autogenerate this header so I don't have to worry about this anymore haha.

Reply:
I'll move it anyways so its in the expected place at the top of the file.